src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -50,6 +62,19 @@
 			class="app-settings-section">
 			<MatterbridgeSettings />
 		</AppSettingsSection>
+		<!-- Destructive actions -->
+		<AppSettingsSection
+			v-if="canLeaveConversation || canDeleteConversation"
+			:title="t('spreed', 'Danger zone')"
+			class="app-settings-section"
+			:token="token"
+			:can-leave-conversation="canLeaveConversation"
+			:can-delete-conversation="canDeleteConversation">
```

Comment:
I'm pretty sure `AppSettingsSection` doesn't have those props
```suggestion
			class="app-settings-section">
```